.github/workflows/tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Test
+
+on:
+  push:
+    paths-ignore:
+      - "layouts/**"
+      - "*.md"
+      - "*.rst"
+
+  pull_request:
+    paths-ignore:
+      - "layouts/**"
+      - "*.md"
+      - "*.rst"
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    # We want to run on external PRs, but not on our own internal PRs as they'll be run
+    # by the push to the branch. Without this if check, checks are duplicated since
+    # internal PRs match both the push and pull_request events.
+    if:
+      github.event_name == 'push' || github.event.pull_request.head.repo.full_name !=
+      github.repository
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install ".[dev]"
+
+      - name: Run tests
+        run: |
+          python -m kalamine.cli layouts/*.toml
```

Comment:
I’m afraid I’ve just broken this with #74 and I’m not sure how to fix it.

I’m now using this line in the Makefile (and I’m not sure it’s correct) : 
```bash
kalamine make layouts/*.toml
```

What’s the proper way to do it ?